R/merge.R

```diff
@@ -196,24 +196,24 @@
     seq
 }
 
-.merge_rows_TSE <- function(x, f, archetype = 1L, mergeTree = FALSE,
-                            mergeRefSeq = FALSE, ...){
+.merge_rows_TSE <- function(x, f, archetype = 1L, agglomerate.tree = FALSE,
+                            agglomerate.refseq = FALSE, ...){
```

Comment:
Have you taken into account deprecation (it might that old names are not needed here if the function that is calling these is taken care of that) --> check

Reply:
yes, changed old names in internal functions because i deprecated in functions that call the.